src/openrct2/rct2/T6Exporter.cpp

```diff
@@ -108,7 +108,7 @@ bool T6Exporter::SaveTrack(IStream* stream)
         for (const auto& entranceElement : _trackDesign->entrance_elements)
         {
             tempStream.WriteValue<uint8_t>(entranceElement.z);
-            tempStream.WriteValue<uint8_t>(entranceElement.direction);
+            tempStream.WriteValue<uint8_t>(entranceElement.direction | (entranceElement.isExit << 7));
```

Comment:
I have doubts. I think this code does not work correctly. If the `highest bit entranceElement.direction == 1` and `entranceElement.isExit == 0`, then the highest bit will be incorrect. We always want to fill it with the value `entranceElement.isExit`?

Reply:
Direction is always a value 0-3. The higher bits are never set.